1-exercises/D-do-while/exercise.js

```diff
@@ -7,7 +7,13 @@
 */
 
 function evenNumbersSum(n) {
-    // TODO
+// TODO
+ let i = 0;
+ let sum = n * (n - 1);
+ do {
+   return sum;
```

Comment:
This is returning instantly from the evenNumbersSum function, so it only ever executes once. Line 12 (`let sum`) is working out the total on its own, so there's no need for the loop. Is there a way to work out the total by adding to the total on each iteration of the loop?